RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -557,23 +564,15 @@ void PulseShapeFitOOTPileupCorrection::phase1Apply(const HBHEChannelInfo& channe
 
     // Photo statistics uncertainties
     //      sigmaFC/FC = 1/sqrt(Ne);
+    // Note2. (from kPedro): the output number of photoelectrons after smearing is treated very differently for SiPMs: *each* pe is assigned a different time based on a random generation from the Y11 pulse plus the SimHit time. In HPDs, the overall pulse is shaped all at once using just the SimHit time.
+
     noisePHArr[ip] = 0;
     if(channelData.hasTimeInfo() && (charge-ped)>channelData.tsPedestalWidth(ip)) {
       noisePHArr[ip] = sqrt((charge-ped)*channelData.fcByPE());
     }
-
-    /*
-    // FIXME: in the future switch the photostatistics on also for the HPD
-    if(!channelData.hasTimeInfo() && (charge-ped)>channelData.tsPedestalWidth(ip)) {
-
-      // Note1. the fcByPE is not in the DB, but is hard coded in python
-      // https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_0/SimCalorimetry/HcalSimProducers/python/hcalSimParameters_cfi.py#L62
-
-      // Note2. (from kPedro): the output number of photoelectrons after smearing is treated very differently for SiPMs: *each* pe is assigned a different time based on a random generation from the Y11 pulse plus the SimHit time. In HPDs, the overall pulse is shaped all at once using just the SimHit time.
-
-      noisePHArr[ip] = sqrt((charge-ped)*0.3305);
+    if(!channelData.hasTimeInfo() && (charge-ped)>noise_) {
+      noisePHArr[ip] = sqrt((charge-ped)*channelData.fcByPE());
```

Comment:
with the new GT, for HPD,  printing on the screen the channelData.fcByPE() is 0.3305